comeet/Data/Extensions/Date+API.swift

```diff
@@ -12,24 +12,28 @@ extension Date {
     
     static func fromAPI(stringDate: String) -> Date? {
         let formatter = DateFormatter()
+        formatter.timeZone = TimeZone(abbreviation: "UTC")
         formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss-SSSS"
```

Comment:
http://stackoverflow.com/questions/28016578/swift-how-to-create-a-date-time-stamp-and-format-as-iso-8601-rfc-3339-utc-tim